portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,88 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Hisao's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div id="content">
+    <!--Header & Basics-->
+    <div id="info">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Jaime Hisao Yesaki Hinojosa</h1>
     </div>
-  </body>
-</html>
+    <!--Image of Myself-->
+    <div class="imgwrapper">
+      <div class="imgcontainer" id="myimage">
+        <a href="https://www.instagram.com/jaimehisao" target="_blank"><img class="imglink" src="/images/hisao2.jpg"
+            alt="Hisao and Ely" width="300" height="400"></img></a>
+      </div>
+      <!--Dev Social Media Icons & Links-->
+      <div class="row">
+        <div class="column">
+        </div>
+        <div class="imgwrapper" id="git">
+          <div class="imgcontainer">
+            <a href="https://www.github.com/jaimehisao" target="_blank"><img class="imglink" src="/images/octocat.svg"
+                alt="El Logo de Github" width="40" height="70"></img></a>
+          </div>
+        </div>
+        <!--
+        <div class="column">
+          <div class="imgcontainer">
+            <a href="https://mx.linkedin.com/in/jaime-hisao-yesaki-750a02155" target="_blank"><img class="imglink"
+                src="/images/lin.png" alt="Logo de LinkedIn" width="60" height="60"></img></a>
+          </div>
+        </div>
+        -->
```

Comment:
nit: can this be removed?

Reply:
Was planning on implementing the LinkedIn button, but it messed up my alignment. I was planning on implementing it tomorrow, but submit my progress. This also relates to @anniefu 's comment about on the CSS file